app/connectors/GetBusinessDetailsConnector.scala

```diff
@@ -30,19 +30,14 @@ class GetBusinessDetailsConnector @Inject()(val http: HttpClient,
                                             val appConfig: MicroserviceAppConfig
                                            )(implicit ec: ExecutionContext) extends RawResponseReads {
 
-  def getBusinessDetailsUrl(nino: String): String = {
-    val platformUrl = if (appConfig.useBusinessDetailsIFPlatform) appConfig.ifUrl else appConfig.desUrl
-    s"${platformUrl}/registration/business-details/nino/$nino"
-  }
+  def getBusinessDetailsUrl(nino: String): String = s"${appConfig.ifUrl}/registration/business-details/nino/$nino"
 
-  def headers: Seq[(String, String)] = {
-    if (appConfig.useBusinessDetailsIFPlatform) appConfig.ifAuthHeaders else appConfig.desAuthHeaders
-  }
+  def headers: Seq[(String, String)] = appConfig.ifAuthHeaders
 
   def getBusinessDetails(nino: String)(implicit headerCarrier: HeaderCarrier): Future[IncomeSourceDetailsResponseModel] = {
 
     val url = getBusinessDetailsUrl(nino)
-    val jsonReads = if (appConfig.useBusinessDetailsIFPlatform) IncomeSourceDetailsModel.ifReads else IncomeSourceDetailsModel.desReads
+    val jsonReads = IncomeSourceDetailsModel.ifReads
 
     logger.debug("[GetBusinessDetailsConnector][getBusinessDetails] - " +
       s"Calling GET $url \n\nHeaders: $headerCarrier \nAuth Headers: ${appConfig.desAuthHeaders}")
```

Comment:
We have des auth headers here. Perhaps this should be removed? Is there any DES 1171 specific things in appConfig that should be removed also?

Reply:
Line 52 also references DES

Reply:
The des Header are used in other connector like  calculation, charge history, etc.
 will fix the des refs. nice spot 👍